components/page/Footer.tsx

```diff
@@ -35,23 +35,32 @@ const Footer = () => (
             <a
               target="_blank"
               rel="noreferrer"
-              href="https://www.ibm.com/us-en/"
+              href="https://www.redhat.com/en"
             >
-              IBM
+              Red Hat
```

Comment:
Instead of text for the contributors, can we use company logos instead? I am fine with leaving it as a text for the first version and change them to logos in a different PR.

Reply:
This can be added in a future PR.